environments/servlet/core/src/main/java/org/jboss/weld/environment/servlet/WeldServletLifecycle.java

```diff
@@ -64,7 +65,6 @@
 import org.jboss.weld.environment.servlet.deployment.WebAppBeanArchiveScanner;
 import org.jboss.weld.environment.servlet.logging.WeldServletLogger;
 import org.jboss.weld.environment.servlet.services.ServletResourceInjectionServices;
-import org.jboss.weld.environment.servlet.util.ServiceLoader;
```

Comment:
Hm, I think you cannot just replace the import - it works with `Iterable<Container>` and not `Iterable<Metadata<Container>>`.